### PR TITLE
Feature/multithreaded scrapers

### PR DESCRIPTION
**Pull request by davidemarcoli**
# Pull Request Check List

Resolves: #issue-number-here

- [ ] Added **tests** for changed code.
- [ ] Updated **documentation** for changed code.

### CODE REVIEW

File: backend/program/scrapers/__init__.py [hunk 1]
```diff
@@ -19,32 +21,52 @@ def __init__(self, hash_cache):
...
         self.hash_cache = hash_cache
         self.services = {
-            Annatar: Annatar(self.hash_cache),
-            Torrentio: Torrentio(self.hash_cache),
-            Knightcrawler: Knightcrawler(self.hash_cache),
-            Orionoid: Orionoid(self.hash_cache),
-            Jackett: Jackett(self.hash_cache),
-            TorBoxScraper: TorBoxScraper(self.hash_cache),
-            Mediafusion: Mediafusion(self.hash_cache)
+            "Annatar": Annatar(self.hash_cache),
```
Comment (dreulavelle) [inline backend/program/scrapers/__init__.py:24]: use the class as the key

Comment (davidemarcoli) [reply backend/program/scrapers/__init__.py:24]: But then how can i log the name of it? Is it something like `service.__name__`?

Comment (dreulavelle) [reply backend/program/scrapers/__init__.py:24]: its either `service.__name__` or `service.__class__.__name__ `

I cant remember

**Comment by dreulavelle** [comment]
I switched to this branch to test, but it didn't initiate a scrape of a requested item?

**Pull request closed without merging**